driver-core/src/test/java/com/datastax/driver/core/policies/IdempotenceAwareRetryPolicyIntegrationTest.java

```diff
@@ -34,6 +34,7 @@
 /**
  * Integration test with an IdempotenceAwareRetryPolicy.
  */
+@SuppressWarnings("deprecation")
 public class IdempotenceAwareRetryPolicyIntegrationTest extends AbstractRetryPolicyIntegrationTest {
```

Comment:
Good catch, I will remove it from the 3.0.x and see if this test could be removed altogether in 3.1.x.